src/Interfacer.jl

```diff
@@ -85,7 +90,7 @@ function Base.show(io::IO, sim::CoupledSimulation)
         "Coupled Simulation\n",
         "├── Running on: $(device_type)\n",
         "├── Output folder: $(sim.dirs.output)\n",
-        "└── Current date: $(sim.dates.date[])",
+        "└── Current time (hours): $(sim.t[] / 3600)\n",
```

Comment:
I think the date would be more useful. Think about running for 1 year (h = 8760 ...)